### PR TITLE
feat(marketplace): add @dimano/ts-devtools-plugin-prefetch-heatmap

### PR DESCRIPTION
**Pull request by dimitrianoudi**
## 🎯 Changes

Adds @dimano/ts-devtools-plugin-prefetch-heatmap to the Devtools Marketplace registry.
Title: Prefetch Heatmap
Description: Visualize TanStack Router prefetch wins vs. waste with an overlay + metrics panel.
Requires: @tanstack/react-router >= 1.0.0
Import type: default export (function)
Links: README, repo, npm
Tags: Router, Prefetch, Analytics, Overlay

## ✅ Checklist

- [✓] I have followed the steps in the [Contributing guide](https://github.com/TanStack/config/blob/main/CONTRIBUTING.md).

- [✓] I have tested this code locally with pnpm test:pr.

```bash
pnpm i
pnpm -r build
pnpm test:pr
pnpm dev

```

## 🚀 Release Impact

- [ ] This change affects published code, and I have generated a [changeset](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md).
- [✓] This change is docs/CI/dev-only (no release).

## Registry entry added
```bash
'@dimano/ts-devtools-plugin-prefetch-heatmap': {
  packageName: '@dimano/ts-devtools-plugin-prefetch-heatmap',
  title: 'Prefetch Heatmap',
  description: 'Visualize TanStack Router prefetch wins vs. waste with an overlay and metrics panel.',
  requires: { packageName: '@tanstack/react-router', minVersion: '1.0.0' },
  pluginImport: { importName: 'default', type: 'function' },
  docsUrl: 'https://github.com/dimitrianoudi/tanstack-prefetch-heatmap#readme',
  repoUrl: 'https://github.com/dimitrianoudi/tanstack-prefetch-heatmap',
  author: 'Dimano',
  framework: 'react',
  tags: ['Router', 'Prefetch', 'Analytics', 'Overlay'],
}
```

## Links

Plugin (npm): https://www.npmjs.com/package/@dimano/ts-devtools-plugin-prefetch-heatmap
Reporter (npm): https://www.npmjs.com/package/@dimano/tsr-prefetch-reporter
Repo: https://github.com/dimitrianoudi/tanstack-prefetch-heatmap
README: https://github.com/dimitrianoudi/tanstack-prefetch-heatmap#readme

### CODE REVIEW

**Comment by dimitrianoudi** [comment]
Updated with pluginId + logoUrl; ready for review.”

File: packages/devtools/src/tabs/plugin-registry.ts [hunk 1]
```diff
@@ -212,6 +212,37 @@ const PLUGIN_REGISTRY: Record<string, PluginMetadata> = {
...
+    title: 'Prefetch Heatmap',
+    description:
+      'Visualize TanStack Router prefetch intent, hits, and waste with a color overlay and a live metrics panel.',
+    requires: {
+      packageName: '@tanstack/react-router',
+      minVersion: '1.0.0',
+    },
+    // default export registers the plugin
+    pluginImport: {
+      importName: 'default',
```
Comment (AlemTuzlak) [inline packages/devtools/src/tabs/plugin-registry.ts]: hmm did you test it, I can't remember if I added an edge case to cover the default import?
